playbooks/operationalizing/cloudforms_networking.adoc

```diff
@@ -0,0 +1,105 @@
+
+
+==== SSL Considerations
+
+To provide the greatest level of flexibility, SSL can be configured in several places depending on the architecture of the environment. SSL termination for the API can occur at eiter the LTM or OpenShift level. For the metrics component, SSL termination can either occur at the LTM layer, Hawkular layer or at the HAProxy layer with the help of SSL reencryption support. The following links provide examples of how each type are configured:
+
+* link:https://docs.openshift.com/enterprise/3.1/install_config/cluster_metrics.html#metrics-deployer-using-secrets[Metrics Security Configuration]
+* link:https://docs.openshift.com/enterprise/latest/install_config/certificate_customization.html[Custom OpenShift certificates]
+* link:https://docs.openshift.com/enterprise/3.1/architecture/core_concepts/routes.html#secured-routes[OpenShift router reencryption]
+
+
+=== OpenShift Configuration
+
+NOTE: This section assumes OpenShift cluster metrics has been configured according to the link:https://docs.openshift.com/enterprise/latest/install_config/cluster_metrics.html[Enabling Cluster Metrics documentation]
+
+With the LTM providing the necessary routing of services for CloudForms, there is no longer a requirement for deploying an additional router in OpenShift.
+However, there is one additional configuration item that is required allow CloudForms to access the data produced by Hawkular metrics. Since Hawkular is run as an application in OpenShift, a route must be created using the hostname applied to the virtual servers in the LTM to allow traffic to flow from the VIP through the integrated HAProxy router and on to Hawkular.
+
+Create a route using the following command in the same project containing Hawkular and other metrics resources. The route must be created in the same project as the metrics components in order to properly route requests to the Hawkular service.
+
+[source]
+----
+oc create -n openshift-infra -f - <<METRICS
+apiVersion: v1
+kind: Route
+metadata:
+  name: cloudforms
+spec:
+  host: ose.example.com 
+  to:
+    kind: Service
+    name: hawkular-metrics
+  tls:
+    termination: passthrough
+METRICS
+----
+
+Verify the route was created successfully by running the following command:
+
+[source]
+----
+oc get routes -n openshift-infra cloudforms
+----
+
+A response similar to the following should be returned:
+
+[source]
+----
+NAME         HOST/PORT            PATH      SERVICE            LABELS    INSECURE POLICY   TLS TERMINATION
+cloudforms   ose.example.com             hawkular-metrics                               passthrough
+----
+
+Verify Hawkular can be reached by navigating to the following location in a web browser: https://ose.example.com:5000
```

Comment:
I think we need a closing line at the end about now moving on to normal CloudForms integration. And eventually we will point them back to the main CF doc once published.